USAGE_android.md

```diff
@@ -260,6 +514,215 @@ originally named `gfxrecon_capture.gfxr` and was created at 2:35 PM
+The capture layer will not register its signal handler until the first call to
+`vkMapMemory`.
+As long as the application initializes the crash detection library before
+calling `vkMapMemory`, there should be no conflict.
+
+The conflict occurs when the application initializes its Vulkan component and
+its crash detection library concurrently.
+Applications have been observed to initialize Vulkan and begin uploading
+resources with one or more threads, while at the same time initializing a crash
+detection library from another thread.
+For this scenario, the crash detection library sets its signal handler after the
+first call to `vkMapMemory`, while a resource upload thread is actively writing
+to the mapped memory.
+After the crash detection library sets its signal handler, it immediately
+receives a SIGSEGV event generated by the concurrent write to mapped memory,
+which it detects as a crash and terminates the application.
+
+This may be a problem that is difficult to solve reliably.
+Without the ability to disable the crash detection library, the only option may
+be to add a delay within the first call(s) to vkMapMemory to attempt to force
+the capture library initialization thread to set its signal handler before the
+capture layer does.
+
+#### Memory Tracking Known Issues
+
+There is a known issue with the page guard memory tracking method.
+The logic behind that method is to apply a memory protection to the
+guarded/shadowed regions so that accesses made by the user to trigger a
+segmentation fault which is handled by GFXReconstruct.
+If the access is made by a system call (like `fread()`) then there won't be a
+segmentation fault generated and the function will fail.
+As a result the mapped region will not be updated.
+
+### Troubleshooting Capturing of Applications
+
+There are several steps to take to determine why a capture may not be working.
+Attempt each of these and re-run after the suggested modifications to see if
+the problem resolves.
+
+#### Enable LogCat
+
+First, enable logcat filtering out specifically messages for GfxReconstruct:
+
+```bash
+adb logcat -s gfxrecon
+```
+
+This should report messages that may be useful in determining the source of the
+capture error.
+
+An extended option would be to look for both `gfxrecon` and `vulkan` messages
+using this command:
+
+```bash
+adb logcat -s gfxrecon,vulkan
+```
+
+#### Stop Your App Before Capture
+
+Sometimes, the application could still be running in the background prior to
+enabling the capture layer.
+In this case, it is best to force stop the application and restart it in order
+for the capture to work properly.
+
+```bash
+adb shell am force-stop ${Application Full Name}
+```
+
+**NOTE:** Replace `${Application Full Name}` with the full Android name of the
+application being captured, for example "com.khronos.vulkan_samples" is the full
+name of the Vulkan Samples.
+
+#### Re-installing The Application
```

Comment:
I'm not sure about this whole section.  I've never had to uninstall an app.  Maybe run this by @mikes-lunarg ?

Reply:
@mikes-lunarg  said install works to replace the old files for him.  So I'll strip it.